buildlib/pr/main.yml

```diff
@@ -296,6 +282,46 @@ stages:
             name: gpu
             demands: ucx_gpu -equals yes
 
+  - stage: Build_Static
+    dependsOn: [Static_check]
+    jobs:
+      jobs:
+      - job: build_source
+        pool:
+          name: MLNX
+          demands:
+            - ucx_docker -equals yes
+        strategy:
+          matrix:
+            centos7:
+              CONTAINER: centos7_ib
+              extra_tls: dc_mlx5 rc_mlx5 ud_mlx5 rc_verbs ud_verbs cma
+              run_tls: ib rc rc_v rc_x dc dc_x ud ud_v ud_x shm sm
+            ubuntu2004:
+              CONTAINER: ubuntu2004
+              extra_tls: ""
+              run_tls: ""
+            ubuntu1804:
+              CONTAINER: ubuntu1804
+              extra_tls: ""
+              run_tls: ""
```

Comment:
@hoopoepg do we need to pass shared memory TLs here?

Reply:
yes: empty tls list means that build should complete, but testing should not run (as I remember some docker containers are not configured properly to run tests)